packages/site-kit/src/lib/markdown/utils.js

```diff
@@ -225,7 +223,7 @@ export function transform(markdown, renderer = {}) {
 		}
 	});
 
-	return marked(markdown);
+	return (await marked.parse(markdown)) ?? '';
```

Comment:
Why is this suddenly async now?

Reply:
Cuz it gives error with these typings. We can prolly just work with type casting, but I'm looking into async-ifying more stuff in here, so I'll leave it here for now

Reply:
> I'm looking into async-ifying more stuff in here

Async will actually slow things down if it's not io-bound, if perf is what you're after.

When does it return nullish btw? silently turning it into an empty string seems like a footgun

Reply:
I don't know when it returns nullish. Yep, that's a footgun, but we're already doing that approach quite a bit everywhere. Throwing an error would complicate the entire codebase quite a bit because of the domino effect of this being used everywhere. Previous marked() function's type didn't have `undefined` in it, and that never broke, so I assume this is fine as well